docs/classic.md

```diff
@@ -5,7 +5,7 @@ Without any additional configuration, FrankenPHP operates in classic mode. In th
 Similar to Caddy, FrankenPHP accepts an unlimited number of connections and uses a [fixed number of threads](config.md#caddyfile-config) to serve them. The number of accepted and queued connections is limited only by the available system resources.
 The PHP thread pool operates with a fixed number of threads initialized at startup, comparable to the static mode of PHP-FPM. It's also possible to let threads [scale automatically at runtime](performance.md#max_threads), similar to the dynamic mode of PHP-FPM.
 
-Queued connections will wait indefinitely until a PHP thread is available to serve them. To prevent that, you can use the `max_wait_time` [configuration](config.md#caddyfile-config) to limit how long a request may wait for a free PHP thread before being rejected.
+Queued connections will wait indefinitely until a PHP thread is available to serve them. To avoid this, you can use the max_wait_time [configuration](config.md) in FrankenPHP's global configuration to limit the duration a request can wait for a free PHP thread before being rejected.
```

Comment:
Sorry, you should out back the #caddyfile-config. I didn't check what the headline was called in each language.